pyproject.toml

```diff
@@ -11,25 +11,25 @@ packages = [{ include = "pylenium" }]
 pylenium = "pylenium.scripts.cli:app"
 
 [tool.poetry.dependencies]
-python = "^3.7"
-pydantic = "1.9.0"
-requests = "2.27.1"
-pytest-xdist = "2.4.0"
+python = ">=3.8.1,<4.0"
+pydantic = "1.10.9"
+requests = "2.31.0"
+pytest-xdist = "3.3.1"
```

Comment:
Also here: Why the pin on a single version instead of a semver specifier?

Reply:
There were many dependency conflicts with all of these packages. They were all semver before, but we pinned them to resolve those conflicts. If the latest versions of these packages don't have those conflicts anymore, then we can make them semver again.

reporportal was one of the biggest offenders of this, but I removed it as a dependency, so we might be able to update these too!

Reply:
No more conflicts 🎉 🎉 🎉 

PR created, just waiting for pipeline to pass so I can publish: https://github.com/ElSnoMan/pyleniumio/pull/306